content/acra/configuring-maintaining/general-configuration/acra-addzone.md

```diff
@@ -3,7 +3,7 @@ title: acra-addzone
 weight: 10
 ---
 
-# acra-addzone
+# acra-addzone (deprecated since 0.94.0)
 
 `acra-addzone` is a command-line utility that generates new [Zone keys](/acra/security-controls/zones/) for AcraBlocks/AcraStructs.
```

Comment:
add couple of words about differentiating ClientID dynamically via SQL?

Reply:
not sure that it is good place on acra-addzone page. Because it is small tool for zones generation. If we want to mention this feature, much better to do it on page about zones in general.

Reply:
I agree, we should describe differentiating ClientID somewhere (a page of your choice), and then _mention_ it on pages relevant to Zones. 

Something like "Zones are deprecated since 0.94.0, but if you are looking for a flexible approach of differentiating clients, check the `differentiating ClientID dynamically via SQL` page".

mark + suggest a replacement approach you've described above